examples/python/example_type_awareness.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2016, Cossack Labs Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# coding: utf-8
+import argparse
+import json
+
+from sqlalchemy import (Table, Column, Integer, MetaData, select, LargeBinary, Text, BigInteger)
+from sqlalchemy.dialects import postgresql
+
+from common import get_engine, get_default, register_common_cli_params
+
+metadata = MetaData()
+test_table = Table(
+    'test', MetaData(),
+    Column('id', Integer, primary_key=True, nullable=False),
+    Column('data_str', Text, nullable=True),
+    Column('masking', Text, nullable=True),
+    Column('token_i32', Integer, nullable=True),
+    Column('data_i32', Integer, nullable=True),
+    Column('token_i64', BigInteger, nullable=True),
+    Column('data_i64', BigInteger, nullable=True),
+    Column('token_str', Text, nullable=True),
+    Column('token_bytes', LargeBinary, nullable=True),
+    Column('token_email', Text, nullable=True),
+)
+# _schema_test_table used to generate table in the database with binary column types
+_schema_test_table = Table(
+    'test', metadata,
+    Column('id', Integer, primary_key=True, nullable=False),
+    Column('data_str', LargeBinary, nullable=True),
+    Column('masking', LargeBinary, nullable=True),
+    Column('token_i32', Integer, nullable=True),
+    Column('data_i32', LargeBinary, nullable=True),
+    Column('token_i64', BigInteger, nullable=True),
+    Column('data_i64', LargeBinary, nullable=True),
+    Column('token_str', Text, nullable=True),
+    Column('token_bytes', LargeBinary, nullable=True),
+    Column('token_email', Text, nullable=True),
+)
+
+rotation_test_table = Table(
```

Comment:
do we use it in our TA examples? `email` has LargeBinary and looks like it was used not for TA cases